test/Passes/tile-and-fuse.mlir

```diff
@@ -0,0 +1,100 @@
+// RUN: tpp-opt %s -split-input-file -tile-consumer-and-fuse-producers -cse | FileCheck %s
+
+func.func @matmul_sequence_fusion(%arg0: tensor<32x64xf32>, %arg1: tensor<64x32xf32>,
```

Comment:
I guess we don't fuse these yet - maybe add a simple check or a comment so that the test isn't completely empty